code/part-two/README.md

```diff
@@ -240,22 +240,22 @@ Let's answer these questions for _Cryptomoji_.
 
 ### Encoding Data
 
-For simplicity and familiarity we are going to encode both payloads and state
+For simplicity and familiarity, we are going to encode both payloads and state
 data as JSON. To be clear, JSON would _not_ be a great choice in production. It
-is not space efficient, and worse, it is not _deterministic_. Determinism is
-very important when writing state to the blockchain. Many many nodes will be
-attempting to the write the same state. If that state is even slightly
-different (like say, the keys are in a different order), the validators will
+is not efficient with space; worse, it is not _deterministic_. Determinism
+is very important when writing state to the blockchain. Many many nodes will
+attempt to the write the same state. If that state is even slightly
+different (like, say, the keys are in a different order), the validators will
 think the transactions are invalid.
 
 But JSON is easy and accessible, especially in Javascript. And it is possible
 to [create sorted JSON strings](https://stackoverflow.com/a/16168003/4284401),
-which should solve the determinism issue (at least well enough for our
+which should solve the determinism issue (at least, well enough for our
 purposes). Of course, JSON itself isn't quite enough, because we need _raw
-bytes_ not a string. For byte encoding we are going to use Node's
+bytes_, not a string. For byte encoding, we are going to use Node's
 [Buffers](https://nodejs.org/api/buffer.html) again.
 
-So any sort of encode we do should look something like this:
+So our encoding should look something like this:
```

Comment:
"your encoding" to be consistent with your other changes

Reply:
Good catch - thanks!